src/routes/article.ts

```diff
@@ -0,0 +1,35 @@
+import createHttpError from 'http-errors';
+import { OK } from 'http-status-codes';
+import {
+  DefaultStateExtends, Middleware, Next,
+} from 'koa';
+import url from 'url';
+import { AppContext } from '../app';
+import { namedNode } from '../data-factory';
+import ArticleNotFound from '../errors/article-not-found';
+
+export default (): Middleware<DefaultStateExtends, AppContext> => (
```

Comment:
`AppMiddleware`

Reply:
Happy to do this but we're still left with the `Dependency cycle detected` lint error.